File: conceptos/conditionals.py
# Condicionales
""" 

if condition:
    codigo

if condition:
    condigo
else:
    condigo


if condition:
    pass
elif condition:
    pass
elif condition:
    pass
else:
    pass


variable = value if condition else value2



"""

a = 5 
b = 10
c = 8

if a > b:
    print("A > B")


if a > c:
    print("A > C")
else:
    print("A < C")


if a > b and a > c:
    print("A > B y C")
elif b > c:
    print("B > A y C")
else:
    print("C > A y B")


mayor = True if a > 18 else False

if mayor:
    print("Puedes Pasar")

File: conceptos/conditionals.js
// Condicionales

/* 

if(condition){
codigo;
}

if(condition){
    codigo
} else {
    condigo
}


if(condtion){

} else if (condition){
 
} else {
 
}


variable = condition ? verdadero : falso


switch(variable){
    case value:
        codigo;
        break;

    default: 
        codigo; 
        break;
}


*/

let a = 5;
let b = 10;
let c = 8

if(a > b){
    console.log("A > B")
}


if (a > c){
    console.log("A > C")
} else {
    console.log("A < C")
}


if(a > b && a > c){
    console.log("A > B y C")
} else if (b > c){
    console.log("B > A y C")
} else {
    console.log("C > A y B")
}


mayor = a > 18 ? true : false

if(mayor){
    console.log("Puedes Pasar")
}